docs/sources/setup-grafana/configure-security/configure-scim-provisioning/configure-scim-with-azuread/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+description: Learn how to configure SCIM provisioning with Azure AD in Grafana Enterprise. This guide provides step-by-step instructions for setting up automated user and team management, including enterprise application configuration, service account creation, attribute mapping, and provisioning settings to ensure seamless integration between Azure AD and Grafana.
```

Comment:
So would the user first follow SAML configuration guide, and then this SCIM guide?  We don't mention the Grafana UI here.

It would be useful to add an end to end walkthrough here, from nothing configured, to everything configured.

Reply:
To clarify, do you mean updating the `description` here, or the entire guide?

We are mentioning SAML configuration as a pre-requisite, it's configuration is a different procedure and we typically try to avoid repetition, instead we just reference to the existing documentation elsewhere.